packages/react-app/src/components/pages/Bounties/index.tsx

```diff
@@ -22,6 +22,11 @@ const Bounties = ({ id }: PreFilterProps): JSX.Element => {
 	const [page, setPage] = useState(0);
 	const [status, setStatus] = useState('');
 	const [search, setSearch] = useState('');
+	const [gte, setGte] = useState(0);
+	// how to handle the lte === 0 case?
+	const [lte, setLte] = useState(Infinity);
```

Comment:
can we add the min/max state variables to the `useEffect` that `setPage(0)` on line 58? that way when they change the pagination doesn't weird out?

Reply:
@Oktalize Thank you for reviewing and finding that. On the case for pagination.